libs/ShivaVG/src/shVgu.c

```diff
@@ -29,7 +29,9 @@ static VGUErrorCode shAppend(VGPath path, SHint commSize, const VGubyte *comm,
                              SHint dataSize, const VGfloat *data)
 {
   VGErrorCode err = VG_NO_ERROR;
-  VGPathDatatype type = vgGetParameterf(path, VG_PATH_DATATYPE);
```

Comment:
Yeah this was weirdly done on shivas part. Your fix was causing crashing for me and I believe the actual method should be this:
VGPathDatatype type = VGPathDatatype(vgGetParameteri(path, VG_PATH_DATATYPE));